pytest_dbfixtures/executors/postgresql.py

```diff
@@ -33,7 +33,8 @@ class PostgreSQLExecutor(TCPExecutor):
     """
 
     BASE_PROC_START_COMMAND = """{pg_ctl} start -D {datadir}
-    -o "-F -p {port} -c %s='{unixsocketdir}'" -l {logfile} {startparams}"""
+    -o "-F -p {port} -c log_destination='stderr' -c %s='{unixsocketdir}'"
```

Comment:
@tomislater if i read @brunsgaard rationale correctly it's not for FreeBSD and it makes it same for all systems